Cubical/Categories/Displayed/Constructions/FullSubcategory.agda

```diff
@@ -0,0 +1,65 @@
+-- | Full subcategories (not necessarily prop) viewed as displayed categories.
+{-# OPTIONS --safe #-}
+module Cubical.Categories.Displayed.Constructions.FullSubcategory where
```

Comment:
If we don't like "Full subcategory" we could call this `FullyFaithful` because every displayed cat where `Fst` is fully faithful is equivalent to one of this form. We should also consider removing `Cubical.Categories.Constructions.FullSubcategory` and replacing it by this. But there is some univalent stuff in there now that would have to be ported